ExeterTeam25/group-software-project
# Email needs to be verified to login

To Test:
- pip install django-utils-six
- make migrations and migrate
- signup with your Exeter email (if a user already exists with one delete it)
- you should be redirected to the login page with the message "Please Confirm your email to complete registration."
- you should receive an email with a link
- open the link and you should be redirected to the home page with the message "Your account has been confirmed."



Details:
- added the functionality so that the user must verify their email to register
